packages/caliper-vscode-extension/artifacts/json-schema/hyperledger-caliper-fabric-network-configuration.schema.json

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
For now, let's remove this schema from the PR. Let's focus only on the runtime configuration schema.